mon-pix/tests/acceptance/certification-course_test.js

```diff
@@ -375,6 +375,30 @@ describe('Acceptance | Certification | Certification Course', function () {
           expect(contains('Test terminé !')).to.exist;
         });
       });
+
+      context('when test was ended by supervisor', function () {
+        it('should display "Votre surveillant a mis fin…"', async function () {
+          // given
+          server.create('feature-toggle', { id: 0, isEndTestScreenRemovalEnabled: true });
+          const user = server.create('user', 'withEmail', 'certifiable', { hasSeenOtherChallengesTooltip: true });
+          const certificationCourse = this.server.create('certification-course');
+          this.server.create('assessment', {
+            certificationCourseId: certificationCourse.id,
+            state: 'endedBySupervisor',
+          });
+
+          // when
+          await authenticateByEmail(user);
+          await visit(`/certifications/${certificationCourse.id}/results`);
+
+          // then
+          expect(
+            contains(
+              'Votre surveillant a mis fin à votre test de certification. Vous ne pouvez plus continuer de répondre aux questions.'
+            )
+          ).to.exist;
```

Comment:
Ici est-ce que ça fonctionnerait un "expect contains ... to be true" plutôt que "expect contains ... to exist" ?

Reply:
Du fait du retour de contains (un objet il me semble de tête), on est obligés d'utiliser To.exist au lieu de to.be.true